docs/en/latest/plugins/limit-count.md

```diff
@@ -48,6 +48,7 @@ Limit request rate by a fixed number of requests in a given time window.
 | redis_database      | integer | optional             | 0             | redis_database >= 0                                                                                     | When using the `redis` policy, this property specifies the database you selected of the Redis server, and only for non Redis cluster mode (single instance mode or Redis public cloud service that provides single entry).                                                                               |
 | redis_timeout       | integer | optional             | 1000          | [1,...]                                                                                                 | When using the `redis` policy, this property specifies the timeout in milliseconds of any command submitted to the Redis server.                                                                                                                                                                           |
 | redis_cluster_nodes | array   | optional             |               |                                                                                                         | When using `redis-cluster` policy，This property is a list of addresses of Redis cluster service nodes.                                                                                                                                                                                                    |
+| redis_cluster_name  | string  | optional             |               |                                                                                                         | When using `redis-cluster` policy，This property is the name of Redis cluster service nodes.                                                                                                                                                                                                                   |
```

Comment:
It should be a comma in English, and the sentence does not end. `This` should be lowercase.
```suggestion
| redis_cluster_name  | string  | optional             |               |                                                                                                         | When using `redis-cluster` policy, this property is the name of Redis cluster service nodes.                                                                                                                                                                                                                   |
```